Week_5/speller/dictionary.c

```diff
@@ -0,0 +1,101 @@
+#include <string.h>
+#include <stdbool.h>
+#include <ctype.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <strings.h>
+
+
+unsigned int Count_size = 0;
+
+#include "dictionary.h"
+
+typedef struct node
+{
+    char word[LENGTH + 1];
+    struct node *next;
+}
+node;
+const unsigned int N = 500;
```

Comment:
```suggestion
} node;

const unsigned int N = 500;
```

I would prefer this style :)